deployment/k8s/charts/components/bulkLoad/templates/deployment.yaml

```diff
@@ -34,6 +34,15 @@ spec:
           volumeMounts:
             - name: {{ $envMountName }}
               mountPath: /shared
+            {{- if .Values.snapshotVolumeEnabled }}
```

Comment:
I see checks for the shared log volume and the snapshot volume to be enabled, why are check if they are enabled?  I would assume that if we are deploying any part of MA the shared log volume is always enabled, is that correct?

Reply:
This component chart could potentially be deployed independently, in which case you may want to use something like the local filesystem instead of mounting a separate volume here

Reply:
Can this be written in such a way that this chart doesn't need to know if its a local file system or volume mount?  Seems like a concern outside of this chart.

Reply:
I don't think so, but maybe we're talking past each other. I saw this as leaving the option to mount a snapshot volume or not, in which case it seems like the Chart will need to know whether to mount the volume in the container definition or have no volume mount

Reply:
Maybe I'm thinking the charts works in a way it doesn't - lets chat offline thanks!

Reply:
It's the chart's concern to do this wiring.  If this wasn't enabled, then we'd be using a local FS.
If it wasn't the chart's responsibility to know how to setup the pod and its mounts, whose responsibility would it be?